src/scene/materials/standard-material.js

```diff
@@ -47,6 +47,7 @@ const _propsSet = [];
  * @property {number} diffuseMapUv Main (primary) diffuse map UV channel.
  * @property {Vec2} diffuseMapTiling Controls the 2D tiling of the main (primary) diffuse map.
  * @property {Vec2} diffuseMapOffset Controls the 2D offset of the main (primary) diffuse map. Each component is between 0 and 1.
+ * @property {number} diffuseMapRotation Controls the 2D rotation of the main (primary) diffuse map.
```

Comment:
It should specify degrees vs radians as a unit.

Reply:
ideally it would be degrees I suspect, to be user friendly and consistent with angles in Quat.

Reply:
UI should be concerned with presenting human-readable values to user, not the internal database. I've gone with degrees, but I'm not convinced that's the right call. (For example glTF stores radians). @willeastcott any opinion?

Reply:
Yes I see quaternion `getEulerAngles` returns degrees. This is not great. The last line of the function does `.mulScalar(math.RAD_TO_DEG)` - a completely unnecessary operation if the returned values are used to perform more math (which would require another `mulScalar(DEG_TO_RAD)` before being useful).

So I guess in the name of consistency we should keep degrees here.

Reply:
All other APIs in the engine use degrees. So, like it or not, for consistency, we should go with degrees. If we want to adopt radians across the engine, that's another conversation.